sdk/communication/azure-communication-chat/azure/communication/chat/_chat_thread_client.py

```diff
@@ -280,6 +280,7 @@ def send_message(
         :paramtype chat_message_type: Union[str, ~azure.communication.chat.ChatMessageType]
         :keyword str sender_display_name: The display name of the message sender. This property is used to
             populate sender name for push notifications.
+        :keyword dict[str, str] metadata : Message metadata.
```

Comment:
I don't see any updates to the  SDK version - I presume this will be shipping as a new preview?
Also, could you please add some details to the changelog?